cvinfer/io.py

```diff
@@ -202,9 +204,14 @@ def _next(self):
             # require looping because of frame skipping feature
             while True:
                 is_valid, frame = self._video_handle.read()
+                if not self._first_frame:
```

Comment:
Is this similar to starting self.frame_count = -1 then keep incrementing without having to check for 1st frame?